app/config/settings.py

```diff
@@ -1048,6 +1048,7 @@
             },
             "versions": {
                 "4.0.0": {},
+                "4.0.1": {},
```

Comment:
Klopt het dat hier een version is toegevoegd ipv vervangen?

Reply:
Ja, klopt. Vervangen zou de zaken die nu gebruik maken van versie 4.0.0 kapot maken.